### ISSUE TITLE
[Feature Request] Redesign Color Picker

### ISSUE THREAD

**Issue by emma-15** [olive-editor/olive#914]
**Describe the solution you'd like**
I think olive should redesign their color picker, it kind of reminds me of ms paint.

**Describe alternatives you've considered**
One thing that should be added to the color picker besides redesigning it, would be a "recent colors" list, that could replace the "custom colors" list.

**Image example**
![cp](https://user-images.githubusercontent.com/48891420/57197955-c8d59800-6f32-11e9-9084-aab87029dbbb.png)

**Comment by BrimsonBhin** [comment]
It's ready considered https://github.com/olive-editor/olive/issues/781#issuecomment-481581123

**Comment by sobotka** [comment]
If you see a hex code in a colour picker, it’s a good sign it was slapped together under a system that has deeper issues than how it looks.

Colour picking is quite tricky in a pixel managed pipeline. More core foundational changes will have to happen first, then the UI pixel management can be tackled. Hard to deal with until a fully pixel managed pipeline is in place.

**Comment by itsmattkc** [comment]
Technically a duplicate of #781. Olive just uses the native color dialog the OS provides (so literally the same as mspaint on Windows). This will have to be changed anyway for various other reasons.

**Comment by BrimsonBhin** [comment]
Closing in favor of #781. That issue has more discussion on the same subject
